app/controllers/api/v2/smart_class_parameters_controller.rb

```diff
@@ -39,7 +41,8 @@ def show
         param :description, String, :desc => N_("Description of smart class")
         param :default_value, String, :desc => N_("Value to use when there is no match")
         param :hidden_value, :bool, :desc => N_("When enabled the parameter is hidden in the UI")
-        param :use_puppet_default, :bool, :desc => N_("Do not send this parameter via the ENC. Puppet will use the value defined in the Puppet manifest for this parameter")
+        param :use_puppet_default, :bool, :desc => N_("Deprecated, please use omit")
+        param :omit, :bool, :desc => N_("Foreman will not send this parameter in classification output. Puppet will use the value defined in the Puppet manifest for this parameter")
```

Comment:
The `omit` in general - yes but since this will be just in the api docs for smart_class_parameter and not anywhere generic I'm find with keeping it a little puppet aware even just to not confuse people that are used to `use_puppet_default`.
I didn't comment on the UI helper because we want to reuse it with the same text for smart variables and that actually uses most of the same view but has a separate api controller.